src/contracts/mod.rs

```diff
@@ -64,6 +65,15 @@ pub struct IdentityManager {
 }
 
 impl IdentityManager {
+    // TODO: I don't like these public getters
```

Comment:
Agreed, we can either make the `abi` and `secondary_abis` fields public or create specific functions to abstract logic that interacts with the abis.

Reply:
Yeah, let's leave the TODO for now. I'm not supper happy with this struct nor the `Ethereum` struct so a bigger refactor would be in order I think